src/os/bluestore/aio.cc

```diff
@@ -5,6 +5,16 @@
 
 #if defined(HAVE_LIBAIO)
 
+std::ostream& operator<<(std::ostream& os, const aio_t& aio)
+{
+  unsigned i = 0;
+  os << "aio: ";
+  for (auto& iov : aio.iov) {
+    os << "\n [" << i++ << "] " << iov.iov_base << " " << iov.iov_len;
```

Comment:
Do you think if it makes sense to print iov_base/iov_len in hex?

Reply:
`iov_base` is a pointer so it's printed in hex already. but `iov_len` is a *length*, i am not sure if we should print it in hex. why shall we do so? because hex digits are aligned and look better?

Reply:
everything else in bluestore is in hex (offsets and lengths).  it's way easier to see block alignment etc.  we should do hex here imo

Reply:
thanks @ifed01 and @liewegas fixed and repushed